packages/atomic/src/components/facets-v1/atomic-rating-facet/atomic-rating-range-facet.tsx

```diff
@@ -0,0 +1,244 @@
+import {Component, h, State, Prop, VNode, Host} from '@stencil/core';
```

Comment:
https://stenciljs.com/docs/style-guide#stencil-style-guide
Stencil's guide recommends:
> One component per directory. Though it may make sense to group similar components into the same directory, we've found it's easier to document components when each one has its own directory.

Reply:
I put them in same directory following this comment: https://github.com/coveo/ui-kit/pull/988#discussion_r675090725. 

If I put this in its separate directory, will I also have to create its own `pcss` file? It uses exactly the same as the one for the rating-facet.